apps/tangle-dapp/containers/ManageProfileModalContainer/Independent/IndependentAllocationInput.tsx

```diff
@@ -136,7 +137,7 @@ const IndependentAllocationInput: FC<IndependentAllocationInputProps> = ({
 
             {min !== null ? (
               <Chip color="dark-grey" className="text-mono-0 dark:text-mono-0">
-                {`≥ ${formatTokenBalance(min, false)}`}
+                {`≥ ${formatTokenBalance(min)}`}
```

Comment:
Showing the unit on the dropdown may cause less space and things to break lines

Reply:
I updated the second parameter of the `formatBalance` function to be the token symbol to format, so in this case, since there is no token symbol as the parameter, it will only return the amount.